doc/customization_basic.rst

```diff
@@ -0,0 +1,150 @@
+
+- Nigeria [NG]
+- Benin [BJ]
+- Botswana [BW]
+- Morocco [MA]
+
+The tutorial mode is selectable by placing within the config file:
+
+.. code:: yaml
+
+    tutorial: true
+
+To perform a full simulation it will be necessary to set:
+
+.. code:: yaml
+
+    tutorial: false
+
+
+Specify the country of interest:
+--------------------------------------
+
+Selection of the country of interest it's possible using the "coutries" argument at the top of the config file.
+
+.. code:: yaml
+
+    countries: ["NG"]
+
+
+Please remember,before proceeding, that as has already been mentioned, in case you are running a simulation in tutorial mode you will only be able to select one nation from NG,BJ,BW or MA; 
+and that for realistic case studies it will be necessary to run the tool in full configuration
+
+Configure `coordinates`
+--------------------------
+Currently, the tool allows us to select the coverage area of interest by going to enter the coordinates of the vertices of the rectangular surface we want to be covered.
+At the end of the config.yaml file you will find the microgrid list section where you can go to enter the coordinates. 
+You can take advantage of the webb OpenStreetMap application to generate the coordinates.
+The code offers the possibility of analysing more than one area of interest at a time.
+
+.. code:: yaml
+
+    microgrids_list:
+        your_microgrid:
+            lon_max: 41.1141
+            lon_min: 41.1086
+            lat_min: -2.0596
+            lat_max: -2.0526
+
+
+Configure ``enable`` section to download/build data
+---------------------------------------------------------
+The tool bases its operation on specific data obtained from open-source platforms. 
+For a successful model run, ensure the download of essential open-source data, including databundle and cost data, is activated in the ``enable`` section:
+
+.. code:: yaml
+
+    enable:
+        retrieve_databundle: true  # Recommended 'true', for the first run. Otherwise data might be missing.
+        retrieve_cost_data: true   # If true, it retrieves cost data from technology data and saves in resources/costs.csv, if false uses cost data in data/costs.csv
+        download_osm_data: true  # If true, OpenStreetMap data will be downloaded for the selected countries
+        download_osm_buildings: true  # If true, OpenStreetMap buildings will be downloaded for the selected countries
+        download_osm_method: overpass # or earth_osm
+        build_cutout: false
+        build_natura_raster: false  # If True, than an exclusion raster will be build
+        
+        # If "build_cutout" : true, then environmental data is extracted according to `snapshots` date range and `countries`
+          # prepare_links_p_nom: false
+
+
+Regarding topics in the enable section:
+
+- Retrive_databundle and retrive_cost_data: After the initial run, it is a good idea to set retrive databundle and cost data to false to avoid re-downloading unnecessary data.
```

Comment:
when referencing option names e.g. "retrieve_databundle", it would be great to surround them with the ` characters to improve the styiling, such as `retrieve_databundle`